### ISSUE TITLE
Pixel mode breaks previous assumptions about layers

### ISSUE THREAD

**Issue by seisatsu** [seisatsu/Driftwood#137]
http://i.imgur.com/5Cm8A4V.png

The top of this bush should be above the player's feet, but since it's on the same layer as them they stand on it. If it were put a layer higher, their head would be under it if they approached from the bottom. Huge oversight. I'm not sure how best to fix this.

**Comment by seisatsu** [comment]
One idea: add a flag to the tile graphic to let it be "tall". Then, the top half of the graphic is drawn after the entities on that layer.

**Comment by pmer** [comment]
@seisatsu Will that work if you come at the plant from the left or right side?

**Comment by seisatsu** [comment]
Hmm, no it won't. Maybe we should do away with the whole colliding with the center of a tile thing. Then support polygons for nowalk in pixel mode.

**Comment by pmer** [comment]
I would like more time to think about this. Removing it from 0.0.6a for now.

**Comment by pmer** [comment]
We could implement #92. That might provide a quicker fix more suitable for 0.0.8.

**Comment by seisatsu** [comment]
How is that?

**Comment by pmer** [comment]
Well, one of the reasons why we added the colliding with the center of a tile thing was to make it easier to walk down 1-tile-wide corridors. But #92 can also solve that problem.

**Comment by seisatsu** [comment]
How will that fix the graphical layering issue?

**Comment by pmer** [comment]
If we remove the colliding with the center of the tile and replace it with colliding at the edge of a tile then entities will no longer appear above tiles on the same layer.

**Comment by seisatsu** [comment]
I don't see what that really has to do with #92 but yes.

**Comment by pmer** [comment]
#92 keeps narrow corridors easy to move through so we don't regress in that area.

**Comment by seisatsu** [comment]
Right, I thought you were saying #92 would fix the graphical issues. We should do both.

**Comment by pmer** [comment]
Depends on #92.
